packages/common/src/ui/components/navigation/NavLink/NavLink.test.tsx

```diff
@@ -74,4 +74,32 @@ describe('NavLink', () => {
     expect(node).toBeInTheDocument();
     expect(node).toHaveAttribute('href', '/customer-invoice');
   });
+
+  it('Correctly renders a span in place of a link for top level hover items', () => {
```

Comment:
very lame test, I was going to get all fancy and add hover checking.. but that was taking a while and I wasn't convinced of the value.